### ISSUE TITLE
Mod doesn't work with diamond or netherite tools

### ISSUE THREAD

**Issue by jsengstock** [ThePotatoArchivist/ToolTrimsMod#3]
Although the mod works with wood, gold, stone, and iron tools, it seems to have no effect when used on diamond or netherite ones. It allows you to put the tool into the smithing table, but doesn't render on the tool.

**Comment by JosiahFu** [comment]
This sounds like a compatability issue. What other mods or resource packs are you using?

**Comment by jsengstock** [comment]
Mods relating to trims: Armor trim item fix, create: armor trims, distracting trims, elytra trims (+ extension), More Armor Trims, Reusable Trims, and this (Tool Trims Mod). I do have about 150 mods total, but a quick scroll through didn't have any obvious culprits jump out at me. Resource packs are Vanilla Evolved and Unique Dark, although I removed them and it didn't seem to fix the issue.

**Comment by JosiahFu** [comment]
Alright thanks, I'll test with those mods and see if I can find the issue

**Comment by JosiahFu** [comment]
Oh also what version are you on?

**Comment by jsengstock** [comment]
1.20.1 -- thanks for investigating!

**Comment by ZettaBytte** [comment]
Hi, same issue here, but is not just diamond and netherite, I've also tried iron and gold. I thought it could be a resource pack thing so I installed the one from the datapack but nothing changed. I realized later that this is a resource pack issue, and one of my packs takes priority over this modded textures, probably because I change the textures for all the tools, but it is weird that the mod doesn't automatically override it. It gets fixed by putting the fabric mod resources at the top of the list, but I think it would be cool if the trims were an overlay like the vanilla trims. It also bothers me to put mod resources at the top because that same pack that changes the tool changes the UI of some mods, but I guess I'll bear with it untill this get's fixed

**Comment by ThePotatoArchivist** [comment]
Ok that must mean some resource pack is overriding the tool models. The issue with making the trims an overlay is that the original textures by JoeFly are not an overlay so I'd have to cut out the trim on every single texture, which I don't think is worth the effort. Also, if a resource pack changes a tool texture it might not match the trim shape, especially the charge trim. What I could do is put all the model overrides in a separate (automatically enabled) resource pack that would let you put just that above other resource packs.

**Comment by ThePotatoArchivist** [comment]
Sorry for no updates on this issue for months, I will try to update it sometime soon

**Comment by ZettaBytte** [comment]
Yeah I thought that could happen. I was thinking what would be worse in the case of a base tool being very altered. If the trim overlay doesn't fit or if it straight up alters the shape. Anyways, if you later review the idea I could remove the took part of the trims. The good part is that, as an overlay, you would only need one texture for, let's say, the emerald frozen trim for the axe, since it would be the same in every axe material.

Anyways, the resource pack solution sounds great, it would be easier to separate between the textures I want to override. Thanks for the attention!

**Comment by ThePotatoArchivist** [comment]
(The trim colors are actually slightly different brightnesses on certain materials, which was another reason I didn't see it as worth it unless I was willing to have disparity with the original textures)

**Comment by ZettaBytte** [comment]
Oooh I didn't noticed. Well, it stays like that, thanks anyways 🙌

**Comment by ThePotatoArchivist** [comment]
The additional resource pack has been added in v1.1.0, make sure to put it above resource packs that modify tool models
